spec/index.html

```diff
@@ -839,32 +841,149 @@ <h3>Terms and Formulae</h3>
            <p>Let <code>T</code>  be the set of <a>N3 triple elements</a> and <code>F</code> the set of <a>N3 formula</a>. We define the function
               <code>nc:T&cup;F&xrarr;T</code> a function defined as follows:
              <l>
-          <li><code>nc(e)=nc(e<sub>1</sub>)&cup;...&cup;nc(e<sub>n</sub>)</code>, if <code>e=e<sub>1</sub>... e<sub>n</sub></code> is an <a>N3 formula</a> and <code>e<sub>1</sub>)&cup;...&cup;nc(e<sub>n</sub></code>
+          <li><code>nc(e)=nc(e<sub>1</sub>)&cup;...&cup; nc(e<sub>n</sub>)</code>,
+            if <code>e=e<sub>1</sub>... e<sub>n</sub></code> is an <a>N3 formula</a> and <code>e<sub>1</sub>...e<sub>n</sub></code>
             are <a>N3 triples</a>,</li>
             <li><code>nc(e)=nc(e<sub>1</sub>)&cup;nc(e<sub>2</sub>)&cup;nc(e<sub>3</sub>)</code>, if <code>e=e<sub>1</sub> c<sub>2</sub> e<sub>3</sub></code> is an <a>N3 triple</a> and
               <code>e<sub>1</sub>, c<sub>2</sub>, e<sub>3</sub></code> are <a>N3 triple elements</a>,</li>
            <li><code>nc(e)=nc(e<sub>1</sub>)&cup;...&cup;nc(e<sub>n</sub>)</code>, if <code>e=(e<sub>1</sub>... e<sub>n</sub>)</code> is a list,</li>
            <li><code>nc(e)=dc(e<sub>1</sub>)&cup; nc(e<sub>1</sub>)</code>,
              if <code>e={e<sub>1</sub>}</code> is a <a>cited formula</a>,</li>
            <li><code>nc(e)=&empty;</code> else.</li>
-
+            </l>
           For each <a>N3 formula</a> <code>f</code>, we call <code>nc(f)</code> the set of <dfn id="nest-comp">nested components</dfn>.
 
 
           </p>
+          <p>
+           We call an <a>N3 formula</a> <dfn data-lt="nesting-free">nesting free</dfn> if it  has only direct components.</p>
+           <p>
+
+      <p>
+       We call an <a>N3 formula</a> <dfn data-lt="ground">ground</dfn> if its direct and nested
+       components do not contain <a>blank nodes</a> nor <a>universal variables</a>.</p>
+       <p>
+        We call an <a>N3 formula</a> <dfn id="uv-free">universal free</dfn> if its direct and nested
+        components do not contain <a>universal variables</a>. We denote the set of universal free quoted graphs by QG.</p>
+
+      <p> We call an <a>N3 formula</a> <dfn data-lt="closed">closed</dfn> if its direct and nested components do not contain <a>universal variables</a>,
+        its direct components do not contain <a>blank nodes</a> and its nested components do only contain blank nodes if they are not subject or object to the predicate
+        <code>log:implies</code>.
+      </p>
+      <p class="ednote">
+        Clarify what "contains" means.
```

Comment:
May be this note dates back to an older version of the text, but I find the term "contains" in this section clear enough. Everytime it is used, its is about a set, and so I interpret it as refering to ∈(element of).

Reply:
thank you for all comments, I just discovered them and will take a look.